repose-aggregator/components/repose-lint/src/main/scala/org/openrepose/lint/commands/VerifyTryItNowCommand.scala

```diff
@@ -487,6 +507,23 @@ object VerifyTryItNowCommand extends Command {
     }
   }
 
+  private def describeFoyerStatus(foyerStatus: FoyerStatus.FoyerStatus): String = {
+    foyerStatus match {
+      case FoyerStatus.Allowed =>
+        "Users with the 'foyer' Identity role WILL pass through this component"
+      case FoyerStatus.AllowedAuthorized =>
+        "Users with the 'foyer' Identity role WILL pass through this component IF AND ONLY IF their Identity " +
+          "service catalog contains an endpoint required by the authorization component"
+      case FoyerStatus.AllowedNotAuthorized =>
+        "Users with the 'foyer' Identity role WILL pass through this component BUT authorization checks will not " +
+          "be performed"
+      case FoyerStatus.NotAllowed =>
+        "Users with the 'foyer' Identity role WILL NOT pass through this component"
+      case FoyerStatus.Unknown =>
+        "Users with the 'foyer' Identity role MAY OR MAY NOT pass through this component"
+    }
+  }
+
   object FoyerStatus extends Enumeration {
     type FoyerStatus = Value
     val Allowed, AllowedAuthorized, AllowedNotAuthorized, NotAllowed, Unknown = Value
```

Comment:
I'm really bad at names.

I like AllowedNoAuthorization better, so I'll make the change. In which case I may also change "AllowedAuthorized" to "AllowedIfAuthorized". Unless you have a better suggestion for that one?